### ISSUE TITLE
Improvement: password on quitting

### ISSUE THREAD

**Issue by SomeDevWeb** [innovation-system/electron-kiosk#10]
Hi all, 
I'm quite interested in a kiosk app and I've actually built one myself last year, but abandoned it at some point cause of lack of time and I got stuck on some more advanced options. I'm not specialized in Electron and my JS knowledge is medium, so my frustration grew exponentially till I left it on a side.
Anyway, this app you made looks quite well integrated and I think the best idea is to work together with others on bigger apps like this, so I'd like to collaborate if you want.
So, back to the point, there are at least 2 options that are absolutely basic and they would need to be implemented in a real production app. 
- One of them is there needs to be **a password when you quit** cause that's the whole point of the app - to keep the user locked into the URL displayed. 
- The other is an optional **top navigation bar** that is enabled/disabled in the options (but that's probably for a separate issue).
I've managed to implement both options and I'll try to retrieve some late version of it from somewhere. 

In the meantime, if you can please reply if any of these options already exists or if there are already plans to work on them?

Thank you very much for your time!

**Comment by Kathou99** [comment]
Yeah is a good options, I can look if I have the time to implement and make a pull request :), I know it's been like 2 years the last release has been done since no more update on it.

**Comment by SomeDevWeb** [comment]
I've tried to work on a similar project, but there are just too many errors or plain electron confusions or deprecations.... :(
This project is the one that makes the most sense and with the newest updates I believe, so I'll try to join it if I have time.
But there are other few crucial feature  requests. Shall I write them here or maybe create independent issues for them?

**Comment by robertsLando** [comment]
@SomeDevWeb Create an issue for each one of them, thanks

**Comment by robertsLando** [comment]
You have to create a fork first, push to your fork and then open a PR
